usr/sbin/p11sak/p11sak.c

```diff
@@ -2566,11 +2567,17 @@ int main(int argc, char *argv[])
     if (!pin) {
         printf("Please enter user PIN:");
         rc = get_pin(&pin, &pinlen);
+        pin_alloced = ckb_true;
```

Comment:
Is 'alloced' an English word?  I would change the variable to e.g. pin_allocated. The code looks good!

Reply:
fixed.